slides.md

```diff
@@ -1,106 +1,134 @@
-class: center, title-page
+---
+theme: slidev-theme-nearform
+layout: image-right
+highlighter: shiki
+lineNumbers: false
+---
+
+<img class=logo src="/images/nearform.svg">
 
 # The Fastify Workshop
 
-<img src="assets/fastify.png" style="width: 30%">
+<img src="assets/fastify.png" style="width: 30%;">
 
----
+<div class="copyright">
+
+© Copyright 2019-2022 NearForm Ltd. All Rights Reserved.
 
-class: branded
+</div>
+
+---
 
 # Introduction: Why Fastify
 
+<div>
```

Comment:
Maybe the `div` is not necessary as well, when slidev parse to HTML, the `div` is automatically added.

Reply:
Good point. I have removed all the divs